layouts/conformance.html

```diff
@@ -0,0 +1,8 @@
+<h1 id="conformance">Conformance</h1>
```

Comment:
this and layouts/conformance.html can be in one file and use <% %> to flex which link is rendered

we should have done this already for medications and medications_r4 if that's what you based this on

Reply:
Made the change here: 
[Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-aef6436aee6121d08a3888a2d722266eL6)

[Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-bfc4783c09a9757e7a3635227cc9c642L5)